x-pack/plugins/apm/public/components/shared/HistoryTabs/__test__/HistoryTabs.test.tsx

```diff
@@ -55,11 +55,11 @@ describe('HistoryTabs', () => {
       }
     ];
 
-    testProps = ({
+    testProps = {
       location: mockLocation,
       history: mockHistory,
       tabs: testTabs
-    } as unknown) as HistoryTabsProps;
+    } as HistoryTabsProps;
```

Comment:
I don't know how you made this work but nice job :) TS kept telling me the types didn't overlap or something, so I had to convert to unknown first.

Reply:
I know! I was having the same problem somewhere else (can't remember where) and just stumbled upon this.

Reply:
Did you make any change that made this work here? Or you just removed the "as unknown" part here and it just worked? Curious just in case I run into that stupid error again

Reply:
No changes as far as I know. Just removed `as unknown` and when ts didn't complain I left it like that